devicon.json

```diff
@@ -4383,7 +4383,32 @@
         "color": "#F18803",
         "aliases": []
     },
-	{
+    {
+        "name": "ohmyzsh",
+        "altnames": [
+            "omz",
+            "omzsh"
```

Comment:
never spaces in names and `ohmyz.sh` is the url of there website nothing to do with the product / tool

Reply:
If you don't let google autocorrect it for you, the first result is actually the github link mentioned above:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/43886029/180981286-2c50a5dc-afaa-4018-8479-f18c76553c37.png">
https://www.google.com/search?rlz=1C1GCEJ_en&sxsrf=ALiCzsYqCZCXJgcRLbdsbAk6f40GLhwC2A:1658830004994&q=omzsh&nfpr=1&sa=X&ved=2ahUKEwjoxsKpp5b5AhWKQfEDHR8BCSEQvgUoAXoECAIQOg&biw=1280&bih=609&dpr=1.5

Reply:
Without autocorrect in USA:
<img width="741" alt="Screenshot 2022-07-26 at 12 09 58" src="https://user-images.githubusercontent.com/17025808/180981734-a3da71b7-6553-40df-8d28-a14e6da8fa83.png">

Reply:
In France:
<img width="731" alt="Screenshot 2022-07-26 at 12 11 35" src="https://user-images.githubusercontent.com/17025808/180981961-30a716a4-3daa-4ab7-8a36-5e020fbefa1f.png">

Reply:
Weird that USA is like that, I wonder why :O
The French version is same as here though. As you can see in the screenshot it says it autocorrected it to `zsh` for you, and is asking if you want to search with `omzsh` instead

Reply:
Hmm, for me it's the same on USA google too. I know google personalizes results ([even in incognito mode](https://www.techradar.com/news/google-search-results-are-personalized-even-when-incognito-study-finds#:~:text=%22In%20fact%2C%20it's%20simply%20not,results%20for%2057%25%20of%20individuals.)), so that might be why?
![image](https://user-images.githubusercontent.com/43886029/180986937-ec6a3be6-d3bd-4074-9dd9-09375657c7bc.png)

<img width="830" alt="image" src="https://user-images.githubusercontent.com/43886029/180986799-779e3324-f566-418e-990e-33fe5b50490d.png">

I tried with duckduckgo too with the exact search `omyzsh` to get an impartial result, and it's showing a blog for oh my zsh as the top result. Looking at the blog it never mentions `omyzsh`. In the comment section of the blog though, there is a comment mentioning `omyzsh` twice, but I assume that's a typo?
https://duckduckgo.com/?q=%22omyzsh%22&va=b&t=hc&ia=web
<img width="620" alt="image" src="https://user-images.githubusercontent.com/43886029/180988737-665d8617-069c-4e1e-b9d4-96c1ef783f92.png">

![image](https://user-images.githubusercontent.com/43886029/180988156-2745dad6-bca2-4a53-a34d-5ea56fdf6aba.png)